content/docs/integrations/broker/index.md

```diff
@@ -27,104 +27,83 @@ The Broker feature is available on our Growth Plan - see pricing page for more i
+#### Roadie Docker Containers
+Roadie provides pre configured docker containers containing pre-configured snyk-broker clients for use with backstage. At time of writing, we are providing a container image for kubernetes and sonarqube.
 
-Alternatively you can deploy the broker application as a container. Below you can find a base Dockerfile that can be used to construct the broker client image. 
+You can configure them with environment variables containing authentication credentials for the brokered service as well as the broker endpoint and token.
 
-<details>
-<summary>Base Dockerfile</summary>
+e.g. To run the kubernetes client you can run the docker container as follows:
 
+```bash
+docker run \
+   -v "$(pwd)"/certs:/var/lib/certs:ro \
+  --env CA_CERT=/var/lib/certs/ca.cert \
+  --env K8S_SERVICE_ACCOUNT_TOKEN=service-account-token \
+  --env BROKER_TOKEN=kube-api \
+  --env BROKER_SERVER_URL=https://<TENANT_NAME>.broker.roadie.so \
+  roadiehq/broker:kubernetes
 ```
-FROM snyk/ubuntu as base
-
-MAINTAINER Roadie
-
-USER root
-
-RUN apt update && apt install -y make python gcc
-
-RUN apt-get update && apt-get install -y ca-certificates
-
-ENV NPM_CONFIG_PREFIX=/home/node/.npm-global
 
-ENV PATH=$PATH:/home/node/.npm-global/bin
+#### Snyk Broker CLI application
 
-RUN npm install --global snyk-broker
+The snyk-broker client is published as an npm package available to be downloaded, installed on run on a machine configured to run nodejs packages. You can install and run it directly or use the npx command:
 
-FROM snyk/ubuntu
-
-ENV PATH=$PATH:/home/node/.npm-global/bin
-
-COPY --from=base /home/node/.npm-global /home/node/.npm-global
-
-COPY --from=base /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
-
-# Don't run as root
-WORKDIR /home/node
-USER node
-
-EXPOSE $PORT
-
-CMD ["broker", "--verbose"]
+```bash
+npm install --global snyk-broker
+BROKER_TOKEN=test \ 
+BROKER_SERVER_URL=https://<TENANT_NAME>.broker.roadie.so \
+broker
 ```
 
-</details>
-
-
-
-**Example Run command**
-
-`docker run --env-file broker-env -v ~/accept.json /home/node/accept.json my-built-broker-client-image`
-
-##### Standard broker environment variables
-
-Broker client expects a standard set of environment variables to run with good configuration. In the example above we mounted and environment file called `broker-env`. The values within that file, without additional values that might be defined in individual configuration files, can be found below:
-```
-ACCEPT=/home/node/accept.json
-BROKER_SERVER_URL=https://<my-tenant>.broker.roadie.so
-BROKER_TOKEN=my-broker-client-token
-PORT=7341
+```bash
+BROKER_TOKEN=test \ 
+BROKER_SERVER_URL=https://<TENANT_NAME>.broker.roadie.so \
+npx snyk-broker
 ```
 
+In order to do something meaningful with the snyk broker client you will need to configure and `accept.json` file documented in a section below.
```

Comment:
Check the spelling of snykbrokerclient. There's few different versions of it within this file